tests/container-scripts/integration-tests.sh

```diff
@@ -16,4 +16,9 @@ killall -9 test-runner
 set -e
 
 pushd /gravity/orchestrator/test_runner
-RUST_BACKTRACE=full TEST_TYPE=$TEST_TYPE RUST_LOG=INFO PATH=$PATH:$HOME/.cargo/bin cargo run --release --bin test-runner
+
+if [[ "${USE_LOCAL_ARTIFACTS:-0}" -eq "0" ]]; then
+    RUST_BACKTRACE=full TEST_TYPE=$TEST_TYPE RUST_LOG=INFO PATH=$PATH:$HOME/.cargo/bin cargo run --release --bin test-runner
```

Comment:
you can create a variables like "RUN_ARGS" with the "--release --bin test-runner" or "/gravity/orchestrator/target/release/test-runner" depends on "USE_LOCAL_ARTIFACTS". Same for the next script. To reduce the code duplication.

Reply:
ok

Reply:
I think you may have intended for me to pass the RUN_ARGS between scripts, but for now I am only setting it using USE_LOCAL_ARTIFACTS wherever it is needed. I found that passing it between scripts adds too much complexity and is not worth the hassle.